scenarios/turbine-heat-processor/src/main/dist/bin/run-example

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#!/usr/bin/env bash
+
+flink run -c io.pravega.turbineheatprocessor.TurbineHeatProcessor lib/pravega-flink-examples-0.1.0-SNAPSHOT-all.jar $*
```

Comment:
I believe the version name will be different? `pravega-flink-examples-0.1.0-SNAPSHOT-all.jar`

Reply:
You are right, I changed it.